src/docs/guides/variables.md

```diff
@@ -96,9 +105,29 @@ alt="Screenshot of Variables Pane"
 layout="responsive"
 width={2408} height={1150} quality={100} />
 
+## Sealed Variables
+
+Sealed variables are useful when you want to provide a value to your builds and deployments but not have it be visible in the UI (or via the API).
```

Comment:
How about something like - 

Railway provides the ability to **seal** variable values for extra security.  When a variable is sealed, its value is provided to builds and deployments but is never visible in the UI nor can it be retrieved via the API.